app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,5 @@
 <resources>
     <string name="device_admin_label">onebillion</string>
     <string name="device_admin_description">Admin Device Policies Description</string>
+    <string name="content_extraction" translatable="false">यह प्रक्रिया केवल एक बार की जाएगी</string>
```

Comment:
Add a full stop at the end of Hindi statement

Reply:
I have added it. Commit is pending.